preferences.yaml

```diff
@@ -0,0 +1,76 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Preferences API
+  description: >-
+    The API lets you manage attributes associated with users (user preferences).
+
+    You can get any attribute for any user (read only) according to the rights
+    you owe on the user resource. 
+
+    There rights are set up at moment you've subscribed the service exposed by
+    this API.
+  license:
+    name: MIT
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/users/{fiscal_number}/preferences':
```

Comment:
or perhaps if you want to talk about Preferences (as a collection of attributes) instead of Users you can have `/preferences/{fiscal_number}`